deps.ts

```diff
@@ -4,6 +4,8 @@ export {
   WebSocket,
 } from "https://deno.land/std@0.68.0/ws/mod.ts";
 
-export {
-  equal,
-} from "https://deno.land/std@0.68.0/testing/asserts.ts";
+export { equal } from "https://deno.land/std@0.68.0/testing/asserts.ts";
+
+export { delay } from "https://deno.land/std@0.68.0/async/delay.ts";
```

Comment:
remove it, you are only re-exporting it and importing but never using it. + manually writing this functionality is actually shorter than the import

Reply:
Forgot that... it's from the previous implementation :(